ftest.sh

```diff
@@ -55,6 +55,25 @@ trap 'echo "encountered an unchecked return code, exiting with error"' ERR
 IFS=" " read -r -a nodes <<< "${2//,/ }"
 TEST_NODES=$(IFS=","; echo "${nodes[*]:1:8}")
 
+# For nodes that are only rebooted between CI nodes left over mounts
+# need to be cleaned up.
+pre_clean () {
+    i=5
+    while [ $i -gt 0 ]; do
+        if clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh \
+             -S -w "$(IFS=','; echo "${nodes[*]}")" "set -x -e
+             mapfile -t ftest_mounts < <(grep 'added by ftest.sh' /etc/fstab)
+             for n_mnt in \"\${ftest_mounts[@]}\"; do
+                mpnt=(\${n_mnt})
+                sudo umount \${mpnt[1]}
+             done
+             sudo sed -i -e \"/added by ftest.sh/d\" /etc/fstab"; then
+            break
```

Comment:
Looks like this needs an extra leading space.  Or the lines above it need one removed.

Reply:
The break statement should be indented differently than the long if condition since it follows the then.
So the options are to use an two space indent, a 5 space indent or a 6 space indent if you want indents to be a multiple of two.
Updating to a 6 space indent.